src/Model/JoinLinkTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\data\Model;
+
+trait JoinLinkTrait
+{
+    /**
+     * The short name of the join link.
+     *
+     * @var string
+     */
+    protected $join;
```

Comment:
private

Reply:
Renaming would be BC break

Reply:
marking with BC break label, breaking anyway, as previously we assigned an object instead of string